src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -203,11 +203,11 @@ public CommandState GetCommandState(InsertCommentCommandArgs args)
             var service = document.GetRequiredLanguageService<IDocumentationCommentSnippetService>();
 
             var isValidTargetMember = false;
-            _waitIndicator.Wait("IntelliSense", allowCancel: true, action: c =>
+            _uiThreadOperationExecutor.Execute("IntelliSense", defaultDescription: "", allowCancellation: true, showProgress: false, action: c =>
             {
-                var syntaxTree = document.GetRequiredSyntaxTreeSynchronously(c.CancellationToken);
-                var text = syntaxTree.GetText(c.CancellationToken);
-                isValidTargetMember = service.IsValidTargetMember(syntaxTree, text, caretPosition, c.CancellationToken);
+                var syntaxTree = document.GetRequiredSyntaxTreeSynchronously(c.UserCancellationToken);
+                var text = syntaxTree.GetText(c.UserCancellationToken);
```

Comment:
looking at all this code, i don't think naming this .UserCancellationToken adds any value.  Why not just c.CancellationTOken?

Reply:
That's the name of the public API here -- not our choosing.